test/dotnet-watch.Tests/Watch/ProgramTests.cs

```diff
@@ -14,7 +14,7 @@ public ProgramTests(ITestOutputHelper logger)
         {
         }
 
-        [PlatformSpecificFact(TestPlatforms.Windows | TestPlatforms.Linux, Skip = "https://github.com/dotnet/aspnetcore/issues/23394")]
+        [PlatformSpecificFact(TestPlatforms.Windows | TestPlatforms.Linux)]
```

Comment:
This unskip did not work. Details explained here: https://github.com/dotnet/aspnetcore/issues/23394#issuecomment-2181542623